geniza/annotations/models.py

```diff
@@ -247,6 +249,14 @@ def sanitize_html(cls, html):
         else:
             return cleaned_html
 
+    @property
+    def etag(self):
+        """Compute and return an md5 hash of content to use as an ETag"""
+        # must be a string encoded as utf-8 to compute md5 hash
+        content_str = json.dumps(self.content, sort_keys=True).encode("utf-8")
```

Comment:
is hashing `content` sufficient? is there anything stored outside of the content that should invalidate the etag?

Reply:
Nothing else can be changed by the editor, except for `modified`, which shouldn't invalidate the etag if the content doesn't also change. (I think `canonical`, `via`, and `footnote` are theoretically mutable, but can't be changed by the editor and would require database access or the django console to change.)

Reply:
That makes sense! Thanks for confirming. Worth noting this in a comment in the code somewhere? (worth noting somewhere to keep an eye on this in case it ever changes / try to keep it from changing)